technische-universitat-dresden-medizin.csl

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Technische Universität Dresden - Medizin</title>
```

Comment:
Is this a German (only) style? If yes, then you should include the `default-locale` attribute in the <a href="http://docs.citationstyles.org/en/stable/specification.html#the-root-element-cs-style">style node</a> and the title should contain a `(German)` at the end.